README.md

```diff
@@ -153,13 +227,48 @@ import { clearError } from '@americanexpress/one-app-ducks';
 store.dispatch(clearError());
 ```
 
-### `errorReporting`
+### `errorReporting` duck
 
-#### `addErrorToReport`
+The error reporting duck is for sending errors to the `errorReportingUrl` by the environment
```

Comment:
what about "The error reporting duck is for sending client side errors to the `errorReportingUrl` configured by the ..."